client/my-sites/site-monitoring/components/site-monitoring-tab-panel/index.tsx

```diff
@@ -12,29 +15,36 @@ export const tabs = [
 ];
 
 interface SiteMonitoringTabPanelProps {
-	children: React.ComponentProps< typeof TabPanel >[ 'children' ];
 	selectedTab?: SiteMonitoringTab;
 	className?: string;
-	onSelected?: ( tabName: SiteMonitoringTab ) => void;
 }
 
 export const SiteMonitoringTabPanel = ( {
-	children: renderContents,
 	selectedTab = 'metrics',
 	className,
-	onSelected,
 }: SiteMonitoringTabPanelProps ) => {
+	const siteSlug = useSelector( getSelectedSiteSlug ) as string;
 	return (
-		<TabPanel
-			initialTabName={ selectedTab }
+		<SectionNav
+			selectedText={ selectedTab }
 			className={ classnames( 'site-monitoring-tab-panel', className ) }
-			tabs={ tabs }
-			onSelect={ ( tabName ) => {
-				updatePageQueryParam( tabName as SiteMonitoringTab );
-				onSelected?.( tabName as SiteMonitoringTab );
-			} }
 		>
-			{ ( tab ) => renderContents( tab ) }
-		</TabPanel>
+			<NavTabs>
+				{ tabs.map( ( { name, title } ) => {
+					return (
+						<NavItem
+							key={ name }
+							path={
+								name === 'metrics'
+									? `/site-monitoring/${ siteSlug }`
+									: `/site-monitoring/${ siteSlug }/${ name }`
```

Comment:
The previous strategy of calling `updatePageQueryParam()` on click meant we were retaining existing query params as the user went from tab to tab. This is good because if the user filtered the PHP logs to a specific time range, then when they switched to Web logs they would keep that time range they selected.

I think we should keep that behaviour, and we can still do it with this new strategy of using real `<a>` elements.

Something like:

```suggestion
									: `/site-monitoring/${ siteSlug }/${ name }${ new URL( window.location.href ).search }`
```

That might cause issues because we're accessing `window` directly during render (like maybe unit tests will fail?) but hopefully you get the idea.

Reply:
`updatePageQueryParam` did almost the same thing as you described ( `new URL...` ), so I don't think there will be any problems.
Thanks for that!

Reply:
This keeps the query param around when switching to the metrics view too. I'm kind of agnostic to that. I guess it means users won't lose their selection as the jump around the site monitors screens. But I also know one of @danielbachhuber's goals was to keep the URL as pretty as possible :)

It can probably be merged as is and then removed from the metrics view later.